lib/targets/ethers/generation.ts

```diff
@@ -111,6 +112,7 @@ function generateInputType(evmType: EvmType): string {
     case AddressType:
       return "string";
     case BytesType:
+    case DynamicBytesType:
```

Comment:
I think the data type for `bytes` should be `string | Buffer | UInt8Array`, or even just `Buffer | UInt8Array`. In many cases, `bytes` is used to send binary data, for which a `string` is not suitable. @krzkaczor what do you think?

See the comments on issues #98 and #123 for more info.
This comment probably applies to all targets, not only ethers.

Reply:
Thanks for your review @quezak. That makes total sense to me, but this might a bit out of context for issue #98. I'd prefer to open an independent pull request for this change.

Reply:
The original issue was about web3 target so i agree that it doesnt make sense to fix it here 🤷‍♂ I will create a new issue when this is done.